buildSrc/src/main/kotlin/io/spine/dependency/test/JUnit.kt

```diff
@@ -29,36 +29,64 @@ package io.spine.dependency.test
 // https://junit.org/junit5/
 @Suppress("unused", "ConstPropertyName")
 object JUnit {
-    const val version = "5.10.0"
+    const val version = "5.12.2"
+
+    /**
+     * The BOM of JUnit.
+     *
+     * This one should be forced in a project via:
+     *
+     * ```kotlin
+     * dependencies {
+     *     testImplementation(enforcedPlatform(JUnit.bom))
+     * }
+     * ```
```

Comment:
Probably, this piece of docs is no longer relevant.

Reply:
I've added the reference to `BomsPlugin`.